### ISSUE TITLE
Wakeup timer

### ISSUE THREAD

**Issue by Gultak** [msp1974/homeassistant-jlrincontrol#111]
The Jaguar I-Pace provides the possibility to add a Wake-up timer to prevent the car to go into deep sleep after a few days of being idle.

The WattCat JLR 3rd party app is able to set these (the official app seems not to be able to).
Also, jlrpy knows of that function: https://github.com/ardevd/jlrpy/blob/4af112fee5ae39a8f19e0d1bf30da3a6aed02451/jlrpy.py#L546

Can this be added to the integration?

**Comment by msp1974** [comment]
Yes, will look to add this as a service.  As a note, you can set it in the jlr app (or mine can) in vehicle settings.

Edit: and probably a sensor to show what it is set to?

**Comment by Gultak** [comment]
Thanks! And thanks for the pointer, found it in the app :)

And yes, a sensor with the set time would be perfect!
